src/apiClient.js

```diff
@@ -0,0 +1,35 @@
+const apiKey = "8b1e08fa98bbe27c329cdd337e163ed0";
+const language = "en-US";
+const keyAndLanguageParameters = `api_key=${apiKey}&language=${language}`;
+
+const getMovieUrl = id => `https://api.themoviedb.org/3/movie/${id}?${keyAndLanguageParameters}`;
+const getMovieCreditsUrl = id => `https://api.themoviedb.org/3/movie/${id}/credits?${keyAndLanguageParameters}`;
+const getPopularMoviesUrl = page => `https://api.themoviedb.org/3/movie/popular?${keyAndLanguageParameters}&page=${page}`;
+const getSearchMoviesByQueryUrl = (page, query) => `https://api.themoviedb.org/3/search/movie?${keyAndLanguageParameters}&query=${query}&page=${page}`;
+
+const getPersonUrl = id => `https://api.themoviedb.org/3/person/${id}?${keyAndLanguageParameters}`;
+const getPersonCreditsUrl = id => `https://api.themoviedb.org/3/person/${id}/movie_credits?${keyAndLanguageParameters}`;
+const getPopularPeopleUrl = page => `https://api.themoviedb.org/3/person/popular?${keyAndLanguageParameters}&page=${page}`;
+const getSearchPeopleByQueryUrl = (page, query) => `https://api.themoviedb.org/3/search/person?${keyAndLanguageParameters}&query=${query}&page=${page}`;
+
+const getGenresUrl = () => `https://api.themoviedb.org/3/genre/movie/list?${keyAndLanguageParameters}`;
+
+const fetchFromApi = async (url) => {
+    const response = await fetch(url);
+
+    if (!response.ok) {
+        new Error(response.statusText);
+    }
+
+    return await response.json();
+}
```

Comment:
Ja bym tu jeszcze dodał chociażby prostą obsługę błędów z wykorzystaniem mechanizmu try-catch, czyli:

```
const fetchFromApi = async (url) => {
  try {
    const response = await fetch(url);

    if (!response.ok) {
      new Error(response.statusText);
    }
    return await response.json();
  } catch (error) {
    console.error(error);
  }
};
```

Reply:
dobry pomysł, to co prawda dla innych developerów i osób wtajemniczonych bardziej ale nie zaszkodzi jak będzie coś takiego

Reply:
Cofam tamten komentarz, nie potrzeba tutaj bo Saga się tym ma zająć i zajmie :)
kwestia dodania tylko `console.error(error);` do kodu w sadze.
A dodanie tutaj tego "łapania" błędów powodowałoby pewnie dalsze komplikacje bo przy obecnej logice nadawałoby stan undefined dla movies, nie nadawałoby stanu error itd...
Merguje tak jak jest a dopiszę do wyświetlanie błędu w konsoli w sadze przy okazji MoviesPage